ui/src/app/utils.ts

```diff
@@ -136,76 +151,88 @@ export const extractCSR = (csrPemString: string) => {
     }
 }
 
+
 export const extractCert = (certPemString: string) => {
-    if (certPemString == "" || certPemString == "rejected") { return }
+    if (certPemString === "" || certPemString === "rejected") {
+        return null;
+    }
 
-    // Decode PEM to DER
     const binaryDer = pemToArrayBuffer(certPemString);
-
-    // Parse DER encoded certificate
     const asn1 = fromBER(binaryDer);
     if (asn1.offset === -1) {
         throw new Error("Error parsing certificate");
     }
 
-    // Load Certificate object
     const cert = new Certificate({ schema: asn1.result });
```

Comment:
why not `const cert = loadCertificate(certPemString)`? You've already written the function

Reply:
Yup, done. Thanks